101-databricks-workspace-with-customer-managed-keys/prereqs/.settings.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
remove this file and update the sample to handle the KV in a separate RG - that will be the common sceanario.

Reply:
This file I have added to keep Workspace resource in same resource group, otherwise second workspace operation tries to create new workspace which is not I want. We want to update existing workspace and hence added this file.

Reply:
I don't think you need the workspace in the prereq template... but when you add the access policy in the main template you'll need to nest that deployment to the other resourceGroup.  The common pattern in Azure is that vaults are not in the same RG as the resources that use them, so we should provide that guidance in the sample.

Reply:
Ok I am updating the deployment to be all in one template once I change that I will update PR